pages/index.vue

```diff
@@ -521,9 +538,15 @@
                     Für welche Signaturservices kann ich die E-ID einsetzen?
                   </v-expansion-panel-header>
                   <v-expansion-panel-content>
-                    Die E-ID kann für Skribble und andere Signaturservices
-                    eingesetzt werden, die auf Swisscom Signing Service
-                    beruhenden.
+                    Die E-ID kann für
+                    <a
+                      class="link"
+                      href="https://www.skribble.com"
+                      target="_blank"
+                      >Skribble</a
+                    >
+                    und andere Signaturservices eingesetzt werden, die auf
+                    Swisscom Signing Service beruhenden.
```

Comment:
```suggestion
                    dem Swisscom Signing Service beruhen.
```

Now we need Roni or David for this PR as well :-/